common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -218,6 +239,10 @@
               :headers {"Content-Type" "text/html; charset=utf-8"}
               :body (-> resource
                         slurp
+                        (string/replace "%GENERIC-TABLE-OF-CONTENTS-SEARCH-DOCS%" (gconfig/all-generic-table-of-contents gconfig/search-table-of-contents-template))
+                        (string/replace "%GENERIC-TABLE-OF-CONTENTS-INGEST-DOCS%" (gconfig/all-generic-table-of-contents gconfig/ingest-table-of-contents-template))
+                        (string/replace "%GENERIC-SEARCH-DOCS%" (read-generic-markdown "search"))
```

Comment:
This has a code smell. The common lib should not have any reference of search or ingest.

Reply:
Removed all references of search and ingest from common-lib/generics